valhalla/mjolnir/complexrestrictionbuilder.h

```diff
@@ -145,7 +145,8 @@ class ComplexRestrictionBuilder : public baldr::ComplexRestriction {
 
   /**
    * Set the dow mask.  indicates days of week to apply the restriction
-   * @param  dow  day of week - This is a mask (e.g., Mo-Fr = 62).
+   * @param  dow  day of week - This is a mask with first day of week being sunday e.g., Mo-Fr = 62
+   * (111110).
```

Comment:
I think showing 1 bit for each day of week here will be a tad more helpful (`0111110`). Its easy to correlate days to bits then. Also, can we state which bit is Sunday, the MSB or LSB?

Reply:
:+1: @mandeepsandhu Updated